doc/man3/OPENSSL_LH_COMPFUNC.pod

```diff
@@ -184,6 +184,13 @@ audit/verify and also opens the window of opportunity for stack
 corruption and other hard-to-find bugs.  It also, apparently, violates
 ANSI-C.
 
+The LHASH code is not thread safe. All updating operations must be
+performed under a write lock. All retrieve operations should be performed
+under a read lock, I<unless> accurate usage statistics are desired.
+In which case, a write lock should be used for retreve operations
```

Comment:
typo: retreve -> retrieve

Reply:
Thanks.  Got it correct two lines above :(